### ISSUE TITLE
Error tagging new release

### ISSUE THREAD

**Issue by attobot** [dss-extensions/OpenDSSDirect.jl#41]
The REQUIRE file could not be found.
cc: @kdheepak

**Comment by kdheepak** [comment]
@tshort I've created a `release-v0.5.0` branch off of 488b691659767a68f9292a90f00bd27301d5d68c. However now a `REQUIRE` file needs to be added. I've deleted the previous tags and added a new commit on the [`release-v0.5.0`](https://github.com/dss-extensions/OpenDSSDirect.jl/tree/release-v0.5.0) branch with the `REQUIRE` file generated from the `Project.toml` file using [this script](https://gist.github.com/tpapp/3b6def2b43cfb9566e3048c37aeed6ab) from this [issue](https://github.com/attobot/attobot/issues/50#issuecomment-423963871). Should we 

a) Create a new `v0.5.0` tag on this hanging `release-v0.5.0` branch, cherry pick the REQUIRE commit to `master` and tag a `v0.5.1` commit on the latest master? OR
b) Create a new `v0.5.0` tag on the latest `master`?

Thoughts? If you have no preference I'll do the former.

**Comment by tshort** [comment]
Option a sounds best.

**Comment by kdheepak** [comment]
Tagged `v0.5.0` in 24bde12727c0a8ed40b3e0b11af1da88888b0c7b. The PR on METADATA.jl is [here](https://github.com/JuliaLang/METADATA.jl/pull/21550).

**Comment by kdheepak** [comment]
Shoot, the metadata uuid changed when we added a new [`Project.toml`](https://github.com/dss-extensions/OpenDSSDirect.jl/blob/ab4fb25746ce763f767339884563a9c24f3dea23/Project.toml). 

```
ERROR: LoadError: LoadError: Tagged version 0.5.0 of OpenDSSDirect declares the package UUID to be "e8943089-f152-4198-86b4-1f0d562a6643" in its Project file, which does not match the package's METADATA-compatible UUID "a8b11937-1041-50f2-9818-136bb7a8fb06".
To fix this error, change the UUID to be "a8b11937-1041-50f2-9818-136bb7a8fb06" and retag.
Stacktrace:
 [1] error(::String, ::String, ::String, ::String) at ./error.jl:42
 [2] (::getfield(Main, Symbol("##23#24")){VersionNumber})(::String) at /home/travis/build/JuliaLang/METADATA.jl/.test/check-pr.jl:172
 [3] mktempdir(::getfield(Main, Symbol("##23#24")){VersionNumber}, ::String) at ./file.jl:597
 [4] mktempdir(::Function) at ./file.jl:595
 [5] top-level scope at /home/travis/build/JuliaLang/METADATA.jl/.test/check-pr.jl:159
 [6] include at ./boot.jl:317 [inlined]
 [7] include_relative(::Module, ::String) at ./loading.jl:1038
 [8] include(::Module, ::String) at ./sysimg.jl:29
 [9] include(::String) at ./client.jl:398
 [10] top-level scope at /home/travis/build/JuliaLang/METADATA.jl/.test/METADATA.jl:256
 [11] include at ./boot.jl:317 [inlined]
 [12] include_relative(::Module, ::String) at ./loading.jl:1038
 [13] include(::Module, ::String) at ./sysimg.jl:29
 [14] include(::String) at ./client.jl:398
 [15] top-level scope at none:0
in expression starting at /home/travis/build/JuliaLang/METADATA.jl/.test/check-pr.jl:150
in expression starting at /home/travis/build/JuliaLang/METADATA.jl/.test/METADATA.jl:254
The command "sh .test/ci.sh" exited with 1.
```

And I didn't check that. I'll re-release now.

**Comment by kdheepak** [comment]
Fixed in d73861c.

**Comment by kdheepak** [comment]
v0.5.0 and v0.5.1 have been merged into METADATA.jl.
